shared/teams/team/settings-tab/retention/index.tsx

```diff
@@ -28,105 +28,67 @@ export type Props = {
   onShowWarning: (policy: RetentionPolicy, onConfirm: () => void, onCancel: () => void) => void
 }
 
-type State = {
-  saving: boolean
-  selected: RetentionPolicy
-  items: Kb.MenuItems
-}
-
-type PropsWithOverlay<P> = {} & P & Kb.OverlayParentProps
+const RetentionPicker = (p: Props) => {
```

Comment:
- [ ] particularly this one

Reply:
how this used to work was really complex and racy. instead we just don't set the value unless it changes